signatory-sgx/src/protocol.rs

```diff
@@ -2,7 +2,7 @@ use crate::error::Error;
 use crate::seal_signer::SealedSigner;
 use serde::{Deserialize, Serialize};
 
-pub const ENCRYPTION_REQUEST_SIZE: usize = 1024 * 1; // 1 KB
+pub const ENCRYPTION_REQUEST_SIZE: usize = 1024 * 60; // 60 KB
```

Comment:
why 60kb? Could the wire protocol be made, such that it writes 2-4 bytes (or variable length  integer) first to denote the length of the request, followed by the request, instead of allocating a fixed buffer size?

Reply:
Ok, I was going to use the `\n` as the end stream flag, but I saw the `chain-tx-enclave` used like this, so I wrote the same way.

Reply:
don't take `chain-tx-enclave` as example, it's WIP :)